modules/insights-advisor-recommendations-filters.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="insights-operator-advisor-recommendation-filters_{context}"]
+= Advisor recommendation filters
+
+The Insights advisor service can return a large number of recommendations. To focus on your most critical recommendations, you can apply filters to the https://console.redhat.com/openshift/insights/advisor/recommendations[Advisor recommendations] list to remove low-priority recommendations.
+
+By default, filters are set to only show enabled recommendations that are impacting one or more clusters. To view all or disabled recommendations in the Insights library, you can customize the filters.
+
+To apply a filter, select a filter type and then set its value based on the options that are available in the drop-down list. You can apply multiple filters to the list of recommendations.
+
+You can set the following filter types:
+
+* *Name:* Search for a recommendation by name.
+* *Total risk:* An abstract value derived from the likelihood that the detected issue will negatively affect your infrastructure, and the impact on operations if that were to happen. Select one or more values from *Critical*, *Important*, *Moderate*, and *Low*.
+* *Impact:* The potential impact to the continuity of cluster operations. Select from *Critical*, *High*, *Medium*, and *Low*.
+* *Likelihood:* Filter by the potential for a negative impact to clusters if the recommendation comes to fruition. *Critical*, *High*, *Medium*, or *Low*.
+* *Category:* Select one or more categories of recommendations to concentrate on: *Service Availability*, *Performance*, *Fault Tolerance*, *Security*, *Best Practice*.
+* *Status:* Set to only show *Enabled* recommendations by default, you can set the filter to show disabled recommendations.
+* *Clusters impacted:* Also set by default, you can filter to only recommendations currently impacting one or more clusters, and or show non-impacting recommendations.
```

Comment:
```suggestion
* *Clusters impacted: Filter only those recommendations currently impacting one or more clusters. You can also show non-impacting recommendations. This setting is enabled by default.
```
- [Per the ISG](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-slashes#general-text), do not use the construction "and/or".